models/collectionPoint.js

```diff
@@ -32,7 +32,6 @@ module.exports = (sequelize, DataTypes) => {
   collectionPoint.associate = function (models) {
     // associations can be defined here
     collectionPoint.hasMany(models.patient, {
-      onDelete: "CASCADE",
```

Comment:
why was this removed? is the `CASCADE` here unnecessary?

Reply:
No, functionally it has no impact. I was using this when I was experimenting with cascade deletes on non-paranoid tables.